viewmaster/viewmaster.py

```
@@ -2196,9 +2538,55 @@ def trim_html(html):
+    Initializes a Flask app instance, sets the secret key required for Flask-WTF
+    forms, initializes global resources (logger, holdings paths, page cache),
+    and registers the viewmaster blueprint that handles all routing for PDS3
+    holdings browsing.
+
+    Returns:
+        Flask: Configured Flask application instance with the viewmaster
+            blueprint registered and all global resources initialized.
+    """
+
+    app = Flask(__name__)
+    app.secret_key = "Cassini Grand Finale!"    # needed by flask_wtf
```

Ruff (0.15.5)

[error] Possible hardcoded password assigned to: "secret_key"

(S105)